website/docs/targeting/targeting-overview.md

```diff
@@ -0,0 +1,87 @@
+---
+id: targeting-overview
+title: Targeting Overview
+description: Overview of the ConfigCat targeting feature with examples.
+---
+
+Using this feature you will be able to set different setting values for specific users or groups of users in your application. Typical use cases are:
+- Beta testing
+- A/B testing
+- Phased rollouts, canary releases
+
+## How does it work?
+
+1. On the <a href="https://app.configcat.com" target="_blank">ConfigCat Dashboard</a>, you add [Targeting rules] and/or [Percentage options] to your feature flag.
+2. In your application, you pass a [User Object] to the ConfigCat SDK.
+3. The ConfigCat SDK will use the User Object and the rules defined on the Dashboard to determine whether feature should be enabled for the user.
+
+## What is what?
+*TODO - new screenshot*
+![Targeting Overview](../../static/assets/targeting/targeting-overview/targeting.png)
+
+### Feature Flag / Setting
+
+A *setting* is a set of rules that yields a **boolean**, **string**, **integer** or **double** value, which can be used to configure your application. It is also known as a configuration value, configuration setting, configuration parameter, etc.
+
+A *feature flag* is a setting that yields a **boolean** value, that is used to decide whether an application feature should be turned ON or OFF. It is also known as a feature toggle, feature switch, feature flipper, conditional feature, etc.
+
+:::info
+Throughout this documentation, we will use the term *feature flag* to refer to both feature flags and settings for simplicity, because feature flags are the most common type of settings.
```

Comment:
```suggestion
Throughout this documentation, we generally use the term *feature flag* to refer to both feature flags and settings for simplicity, because feature flags are the most common type of settings.
```